ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5028,6 +5031,7 @@ SSPC2X:
   - supply_air_damper_percentage_command
   - supply_air_flowrate_sensor
   - supply_fan_speed_percentage_command
+  - average_supply_air_static_pressure_sensor
```

Comment:
what is the purpose of having this? is it used in control of the device? if so, how?

Reply:
It will not be used for control. It is only shown for indication in BOS. hence captured in opt_uses.